modules/ml/src/test/java/org/apache/ignite/ml/util/generators/DataStreamGeneratorTest.java

```diff
@@ -158,6 +171,39 @@ public void testAsDatasetBuilder() throws Exception {
         checkDataset(N / 2, b3, v -> (Double)v.label() < 0);
     }
 
+    /** */
+    @Test
+    public void testCacheFilling() {
+        IgniteConfiguration configuration = new IgniteConfiguration().setPeerClassLoadingEnabled(true);
+        String cacheName = "TEST_CACHE";
+        CacheConfiguration<UUID, LabeledVector<Double>> cacheConfiguration =
+            new CacheConfiguration<UUID, LabeledVector<Double>>(cacheName)
+                .setAffinity(new RendezvousAffinityFunction(false, 10));
+        int datasetSize = 5000;
+
+        try (Ignite ignite = Ignition.start(configuration)) {
```

Comment:
If it's okay to start Ignite manually in unit tests? I just thought that unit tests might be run in parallel and in this case we'll get unpredictable behavior.

Reply:
Hm, maybe you're right. 
Okay, I'll move this test to "integration" test and use GridAbstracTest class.